s3watcher/lambda/lib/Upload.js

```diff
@@ -20,10 +18,10 @@ module.exports = {
 
          const uploadedBy = buildUploadedBy(s3Event.path);
 
-         return  {
+         return {
              key: config.apiKey,
              url: config.baseUrl,
-             path: "/images",
+             path: "/imports",
              size: s3Event.size,
```

Comment:
I [does](https://github.com/guardian/grid/blob/master/s3watcher/lambda/lib/Transfer.js#L48), though this PR removes that.

I was thinking its more appropriate for image-loader to log this information now as it makes more sense in the context of that service.

Its minor though.